modules/ipip/datapath_out.c

```diff
@@ -57,6 +57,7 @@ ipip_output_process(struct rte_graph *graph, struct rte_node *node, void **objs,
 		tunnel.len = rte_be_to_cpu_16(inner->total_length);
 		tunnel.vrf_id = iface->vrf_id;
 		tunnel.proto = IPPROTO_IPIP;
+		tunnel.ttl = IPV4_DEFAULT_TTL;
```

Comment:
Strange that this issue is seen only now. Is this with a higher level of optimisations at compilation?

Reply:
No, with the default option.
I don't know why it is triggered only now though. (And only on my system !?)

Reply:
Which gcc, which distro?

Reply:
rhel 9.4
gcc version 11.4.1 20231218 (Red Hat 11.4.1-3) (GCC)

Reply:
Ok, I prefer such a reason :-).